tags/src/components/Tag.tsx

```diff
@@ -0,0 +1,56 @@
+import { FunctionComponent, useEffect, useState } from 'react'
+import { Autocomplete, Chip, TextField } from '@mui/material'
+import { useFieldPlugin } from '../useFieldPlugin'
+
+const Tag: FunctionComponent = () => {
+  const { data, actions } = useFieldPlugin()
+
+  const [initialValue, setInitialValue] = useState<string[]>([])
+
+  useEffect(() => {
+    setInitialValue(parseJsonValue(data.options.initialValue))
+  }, [data.options.initialValue])
+
+  useEffect(() => {
+    actions.setContent(initialValue)
+  }, [initialValue])
+
+  return (
+    <Autocomplete
+      multiple
+      options={[]}
+      value={initialValue}
+      onChange={(e, newValue: string[]) => setInitialValue(newValue)}
+      freeSolo
+      disableClearable
+      renderTags={(value: readonly string[], getTagProps) =>
+        value.map((option: string, index: number) => (
+          // eslint-disable-next-line react/jsx-key
```

Comment:
this is not necessary, but is there a way to fix this and not have the disable comment

Reply:
ok then let's leave it as is :) thank you for describing the issue @demetriusfeijoo